### ISSUE TITLE
Alaris IV Pump

### ISSUE THREAD

**Issue by kroylar** [maxroberts/interfacing-to-hospital-equipment#1]
I just started skimming the Alaris IV pump manual and think that we may need some more information. I know the doctors and nurses are extremely busy, but if you could find out from them what specific alarms need silencing that could be really helpful.

Just looking at this one manual, it looks like alarm limits should be programmed such that alarms don't go off unless someone actually needs to inspect the patient. And alarms can only be silenced for 120 seconds maximum.

Also, for this one, I would think the built-in wireless connectivity would be sufficient to control the entire system.

@maxroberts you've read some of the manual, what are your thoughts on the above?

**Comment by maxroberts** [comment]
Yes, going to try to loop some medical workers into this directly. What do you think is the easiest way for them to include their comments? In the wiki, or in these issues? Definitely not in the READMEs.

Wireless option sounds promising! Haven't had a chance to read this manual yet, but skimming real quick:

"The wireless network card provides wireless communication capability between the Alaris System and Alaris Systems Manager. The combined use of the Alaris System and Alaris Systems Manager is integrated into a facility’s existing network infrastructure.
When enabled, the Alaris Systems Manager allows the exchange of information between the Alaris Systems Manager and the Alaris System. The PC unit can be operated
manually or in concert with the information exchanged with the Alaris Systems Manager. If communication with the wireless network is interrupted (for example, out of range), the Alaris System can be used, as intended, in the manual mode."

So this manager may be a piece of hardware they don't have at the hospital? What's confusing is someone from the VA said they talked to Alaris, and Alaris said they didn't have remote capabilities... but it seems like they do. Miscommunication?
